### ISSUE TITLE
[BUG] Rocket Classifier producing subpar results on Arrow Head data on `sktime 0.36.0`

### ISSUE THREAD

**Issue by rubinglen** [sktime/sktime#7889]
There is a bug with the rocket classifier in version .36.  I'm trying to classify the UCR arrowhead dataset and getting a completely unreasonable result where everything is classified by rocket as a single label, while a simple forest classifier returns expected result with few errors.  

```python
from sktime.classification.kernel_based import RocketClassifier
from sktime.datasets import load_arrow_head
from sklearn.metrics import accuracy_score

X_train, y_train = load_arrow_head(split="train", return_X_y=True)
X_test, y_test = load_arrow_head(split="test", return_X_y=True)

clf = RocketClassifier(num_kernels=2000, random_state=42)
clf.fit(X_train, y_train)

y_pred = clf.predict(X_test)
accuracy = accuracy_score(y_test, y_pred)

print("Predicted labels distribution:", {label: sum(y_pred == label) for label in set(y_pred)})
print("Accuracy:", accuracy)`
```

**Comment by fkiraly** [comment]
@rubinglen, thanks for the report. If you have time, what would be useful is a comparison where you apply the rocket feature transformation `Rocket` manually and then use the classifier manually.

FYI, the `RocketClassifier` is an alias for the following pipeline:

```python
make_pipeline(
            univar_rocket,
            StandardScaler(with_mean=False),
            RidgeClassifierCV(alphas=np.logspace(-3, 3, 10)),
        )
```

**Comment by fkiraly** [comment]
While I agree that only a single label would be a problem, I do not get absurd accuracies, and cannot reproduce your problem. We may have to drill down here.

 on current `main`:
```
Predicted labels distribution: {'1': 59, '2': 72, '0': 44}
Accuracy: 0.8057142857142857
```

on version 0.36.0 release:
```
Predicted labels distribution: {'2': 21, '1': 154}
Accuracy: 0.38857142857142857
```

Although the latter does surprise me. The last changes to relevant files:

`Rocket`: 2024-03
`RocketClassifier`: 2024-06
`BaseClassifier`: 2024-08

Any ideas what might be going on?

**Comment by rubinglen** [comment]
Thanks Franz!
